inno_setup/translations/default.iss

```diff
@@ -0,0 +1,90 @@
+; 5) Translation entries allow us to have input arguments. They are specified as %1, %2, etc.
+
+; For a list of default custom messages (Translated by Inno Setup), see: https://jrsoftware.org/ishelp/index.php?topic=custommessagessection
+; Example: we are using AssocFileExtension from this list
+
+; Custom task descriptions - English (default)
+FeatureEnvironmentDesc=Modify PATH environment variable by prepending the JDK installation directory to the beginning of PATH.
+FeatureJavaHomeDesc=Sets or overrides JAVA_HOME environment variable with the JDK installation directory.
+FeatureOracleJavaSoftDesc=Overwrites Oracle's reg key HKLM\Software\JavaSoft. After uninstallation of %1, Oracle Java needs to be reinstalled to re-create these registry keys.
+FeatureEnvironmentTitle=Modify PATH variable
+FeatureJavaHomeTitle=Set or override JAVA_HOME variable
+FeatureJarFileRunWithTitle=Associate .jar
+FeatureOracleJavaSoftTitle=JavaSoft (Oracle) registry keys
+
+German.FeatureEnvironmentDesc=In die PATH-Umgebungsvariable einfügen.
+German.FeatureJavaHomeDesc=Als JAVA_HOME-Umgebungsvariable verwenden.
+; German.FeatureOracleJavaSoftDesc=Überschreibt Oracles Registrierungsschlüssel HKLM\Software\JavaSoft. Nach der Deinstallation von %1 muss Oracle Java neu installiert werden, um diese Registrierungsschlüssel wiederherzustellen.
+German.FeatureEnvironmentTitle=Zum PATH hinzufügen
+German.FeatureJavaHomeTitle=JAVA_HOME-Variable konfigurieren
+; German.FeatureJarFileRunWithTitle=.jar-Datei verknüpfen
+; German.FeatureOracleJavaSoftTitle=JavaSoft (Oracle) Registrierungsschlüssel
+
+Spanish.FeatureEnvironmentDesc=Añadir a la variable de entorno PATH.
+Spanish.FeatureJavaHomeDesc=Establecer la variable de entorno JAVA_HOME.
+Spanish.FeatureOracleJavaSoftDesc=Sobrescribir las claves de registro HKLM\Software\JavaSoft (Oracle). Si se desinstala %1, la ejecución de Oracle Java desde la ruta "C:\Program Files (x86)\Common Files\Oracle\Java\javapath" no funcionará. Será necesario reinstalarlo.
+Spanish.FeatureEnvironmentTitle=Añadir al PATH
+Spanish.FeatureJavaHomeTitle=Establecer la variable JAVA_HOME
+Spanish.FeatureJarFileRunWithTitle=Asociar .jar
+Spanish.FeatureOracleJavaSoftTitle=Claves de registro JavaSoft (Oracle)
+
+French.FeatureEnvironmentDesc=Ajouter à la variable d'environnement PATH.
+French.FeatureJavaHomeDesc=Définir la variable d'environnement JAVA_HOME.
+French.FeatureOracleJavaSoftDesc=Écrase les clés de registre HKLM\Software\JavaSoft (Oracle). Après la désinstallation d'%1, Oracle Java lancé depuis le PATH "C:\Program Files (x86)\Common Files\Oracle\Java\javapath" ne fonctionne plus. Réinstaller Oracle Java si besoin
+French.FeatureEnvironmentTitle=Ajouter au PATH
+French.FeatureJavaHomeTitle=Définir la variable JAVA_HOME
+French.FeatureJarFileRunWithTitle=Associer les .jar
+French.FeatureOracleJavaSoftTitle=Clés de registre JavaSoft (Oracle)
+
+; Japanese.FeatureEnvironmentDesc=JDKインストールディレクトリをPATHの先頭に追加してPATH環境変数を変更します。

```

Comment:
Commented out why?

Reply:
I left an explanation in comments under the [CustomMessages] section marker. Here, commented messages were done via AI and need to be verified by someone who speaks the language. They represent languages that we are supposed to support, but we do not yet have translated yet (all non-commented messages pulled from the msi translations). @karianna